src/components/HomePage/Filters/Filters.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Select } from './../../../utils'
```

Comment:
why do you put this to **utils**? It will be much cleaner if you will keep your pages in **pages** directory and components in **components** (Select, Input, Button)

Reply:
https://github.com/Azaliyaa/netflix-roulette/pull/1/commits/c9362e737f87aed20a979cc23bf00d54e63dd86d